scripts/openvpn-install.sh

```diff
@@ -0,0 +1,1347 @@
+#!/bin/bash
```

Comment:
Why are we creating a new openvpn-install.sh file which has the same content as the one on https://raw.githubusercontent.com/angristan/openvpn-install/master/openvpn-install.sh?